cargo-nextest/src/dispatch.rs

```diff
@@ -606,7 +606,8 @@ pub struct TestRunnerOpts {
         visible_alias = "jobs",
         value_name = "THREADS",
         conflicts_with_all = &["no-capture", "no-run"],
-        env = "NEXTEST_TEST_THREADS"
+        env = "NEXTEST_TEST_THREADS",
+	allow_hyphen_values = true
```

Comment:
Oh I see what happened here -- `allow_negative_numbers` isn't supported on `Arg` in clap 3 (but is in clap 4 -- https://epage.github.io/blog/2022/09/clap4/#fixing-parsing-for-hyphenated-values). I'll land this and then revert the change.